subt/ros/robot/launch/k2.launch

```diff
@@ -1,96 +1,92 @@
+  </node>
 
-  		<arg name="color_width"         value="848"/>
-  		<arg name="color_height"        value="480"/>
-  		<arg name="enable_color"        value="true"/>
+  <!-- Data stream from ROS to Osgar -->
+  <node name="push_to_osgar" pkg="proxy" type="push.py" ns="osgar/to">
+    <param name="output_scan_channel" value="scan360"/>
+    <param name="output_pointcloud_channel" value="points"/>
+  </node>
 
-  		<arg name="depth_fps"           value="15"/>
-  		<arg name="color_fps"           value="15"/>
-	</include>
-	<node name="dynparamROI" pkg="dynamic_reconfigure" type="dynparam" args="set_from_parameters /camera/stereo_module/auto_exposure_roi">
-                <param name="left" type="double" value="200" />
-                <param name="right" type="double" value="648" />
-                <param name="top" type="double" value="330" />
-                <param name="bottom" type="double" value="479" />
-        </node>
-	<include file="$(find realsense2_camera)/launch/rs_t265.launch" if="$(arg record_sensors)">
-		<arg name="camera"                value="rs_t265"/>
-		<arg name="serial_no" 		value="929122111204"/>
-	</include>
-	
-	<node name="laserscan_kinect" pkg="laserscan_kinect" type="laserscan_kinect" >
-                <remap from="image" to="/camera/depth/image_rect_raw"/>
-                <remap from="camera_info" to="/camera/depth/camera_info"/>
-      		<rosparam command="load" file="$(find robot)/params_k2/depth_scan_params.yaml" />
-        	<remap from="scan" to="/scan"/>
-	</node>
-	<!--node pkg="depthimage_to_laserscan" type="depthimage_to_laserscan" name="depthimage_to_laserscan" >
-                <param name="scan_height" value="3"/>
-                <param name="range_min" value="0.2"/>
-                <param name="output_frame_id" value="base_link/front_laser"/>
-                <remap from="image" to="camera/depth/image_rect_raw" />
-                <remap from="camera_info" to="camera/depth/camera_info" />
-        </node-->
-	<node if="$(arg record_sensors)" pkg="tf" type="static_transform_publisher" name="laserscan" args="0 0 0 0 0 0 camera_link base_link/front_laser 100"/>
-        <node name="cliff_detector" pkg="cliff_detector" type="cliff_detector">
-                <remap from="image" to="/camera/depth/image_rect_raw"/>
-                <remap from="camera_info" to="/camera/depth/camera_info"/>
-                <rosparam command="load" file="$(find robot)/params_k2/cliff_detector_params.yaml" />
-        </node>
-        <!-- TF transformations x y z yaw pitch roll -->
-        <node if="$(arg record_sensors)" pkg="tf" type="static_transform_publisher" name="realsense" args="0 0 0.245 0 0 0 /base_link /camera_link 100"/>
-	<node if="$(arg record_sensors)" pkg="tf" type="static_transform_publisher" name="t265_mounting" args="0 0 0.245 0 0 0 /base_link /rs_t265_link 100"/> 
-</launch>
+  <node name="record" pkg="rosbag" type="record" args="-a -o /tmp/skiddy"/>
```

Comment:
Certainly not "skiddy". Check how much data it produces and you will likely want to disable (=delete) this node for important runs.